metricflow/dataflow/sql_table.py

```diff
@@ -16,18 +16,13 @@ def _from_yaml_value(cls, input: PydanticParseableValueType) -> SqlTable:
         """Parses a SqlTable from string input found in a user-provided model specification
 
         Raises a ValueError on any non-string input, as all user-provided specifications of table identifiers
-        should be strings conforming to the expectations defined in the from_string method. The lone exception
-        is for SqlTable instance inputs, which could arise if we are initializing Materialization objects internally
-        without direct access to user input.
+        should be strings conforming to the expectations defined in the from_string method.
         """
         if isinstance(input, str):
             return SqlTable.from_string(input)
-        elif isinstance(input, SqlTable):
-            # This is internally constructed, pass it through and ignore it in error messaging
-            return input
         else:
             raise ValueError(
-                f"SqlTable inputs from model configs are expected to always be of type string, but got type "
+                f"SqlTable inputs from model configs are expected to always be of type string or dict, but got type "
```

Comment:
It was updated to say `string or dict` but this method will only handle strings, are dicts handled somewhere else?

Reply:
Ahh I see, [metricflow/model/objects/base.py#145](https://github.com/transform-data/metricflow/blob/4536401243884fc45cd970b0b9395b10444d96b0/metricflow/model/objects/base.py#L145) handles the dict case

Reply:
Oh that's a mistake actually, will revert before merge.